source/Patches/Roles/Role.cs

```diff
@@ -488,17 +488,31 @@ public static void Postfix(ref string __result, [HarmonyArgument(0)] StringNames
             {
                 if (ExileController.Instance == null || ExileController.Instance.exiled == null) return;
 
+                // If Lover was Voted and option was activated we display the role and the other lover in the ejection text to prevent other players to replace the body appaerant
+                //anyway we would have understood that they were lovers when the otherlover dies if the option is not activated
+                var info = ExileController.Instance.exiled;
+                var role = GetRole(info.Object);
+                if (role == null) return;
+                var roleName = "";
```

Comment:
Define this inside the `if` block, so you don't end up assigning it twice. Variables should be only assigned once unless there's a need for it to be mutable.